util/importers/fisher.py

```diff
@@ -0,0 +1,313 @@
+            
+            # Open wav corresponding to transcription file
+            wav_filename = os.path.splitext(os.path.basename(trans_file))[0] + ".wav"
+            wav_file = os.path.join(source_dir, wav_filename)
+            
+            print("splitting {} according to {}".format(wav_file, trans_file))
+            
+            origAudio = wave.open(wav_file, "r")
+            
+            # Loop over segments and split wav_file for each segment
+            for segment in segments:
+                # Create wav segment filename
+                start_time = segment["start_time"]
+                stop_time = segment["stop_time"]
+                new_wav_filename = os.path.splitext(os.path.basename(trans_file))[0] + "-" + str(start_time) + "-" + str(stop_time) + ".wav"
+                new_wav_file = os.path.join(target_dir, new_wav_filename)
+                
+                # If the wav segment filename does not exist create it
+                if not os.path.exists(new_wav_file):
+                    _split_wav(origAudio, start_time, stop_time, new_wav_file)
+            
+            # Close origAudio
+            origAudio.close()
+            
+            # Remove wav_file
+            # os.remove(wav_file)
+
+def _split_wav(origAudio, start_time, stop_time, new_wav_file):
+    frameRate = origAudio.getframerate()
+    origAudio.setpos(int(start_time*frameRate))
+    chunkData = origAudio.readframes(int((stop_time - start_time)*frameRate))
+    chunkAudio = wave.open(new_wav_file, "w")
+    chunkAudio.setnchannels(origAudio.getnchannels())
+    chunkAudio.setsampwidth(origAudio.getsampwidth())
+    chunkAudio.setframerate(frameRate)
+    chunkAudio.writeframes(chunkData)
+    chunkAudio.close()
+
+def _maybe_split_transcriptions(data_dir, original_data, converted_data):
+    source_dir = os.path.join(data_dir, original_data)
+    target_dir = os.path.join(data_dir, converted_data)
+    
+    if os.path.exists(os.path.join(source_dir, "split_transcriptions_done")):
+        print("skipping maybe_split_transcriptions")
+        return
+    
+    # Loop over transcription files and split them into individual files for
+    # each utterance
+    for root, dirnames, filenames in os.walk(source_dir):
+        for filename in fnmatch.filter(filenames, "*.txt"):
+            trans_file = os.path.join(root, filename)
+            segments = _parse_transcriptions(trans_file)
+            
+            # Loop over segments and split wav_file for each segment
+            for segment in segments:
+                start_time = segment["start_time"]
+                stop_time = segment["stop_time"]
+                txt_filename = os.path.splitext(os.path.basename(trans_file))[0] + "-" + str(start_time) + "-" + str(stop_time) + ".txt"
+                txt_file = os.path.join(target_dir, txt_filename)
+                
+                transcript = validate_label(segment["transcript"])
```

Comment:
@andrenatal @reuben You will have to work together on `validate_label()` as both your PR's add this function